Core/include/Acts/Surfaces/BoundaryTolerance.hpp

```diff
@@ -108,10 +108,10 @@ class BoundaryTolerance {
         : maxChi2(maxChi2_), weight(weight_) {}
   };
 
-  enum class ToleranceMode {
-    Extend,  // Extend the boundary
-    None,    // No tolerance
-    Shrink   // Shrink the boundary
+  enum class Mode {
+    Extend,  //< Extend the boundary
+    None,    //< No tolerance
+    Shrink   //< Shrink the boundary
   };
```

Comment:
I thought this might be a bit more concise @paulgessinger - but no strong preference on my side

Reply:
Mode might be enough, yeah.